dlt/cli/pipeline_command.py

```diff
@@ -114,9 +114,15 @@ def _display_pending_packages() -> Tuple[Sequence[str], Sequence[str]]:
 
     if operation == "show":
         if command_kwargs.get("marimo"):
+            import os
+
+            from dlt.common.utils import custom_environ
+            from dlt.common.known_env import DLT_DATA_DIR
+
             from dlt.helpers.studio.runner import run_studio
 
-            run_studio()
+            with custom_environ({DLT_DATA_DIR: os.path.dirname(p.pipelines_dir)}):
```

Comment:
does this work if we start marimo with `subprocess.run`? I thought I had tested this and the env set in python does not get carried over to other processes.

Reply:
hmmm AFAIK env is carried over. I have tests for that. maybe you should use our `Venv.current()` to start processes. then it is carried over.